### PR TITLE
feat(client): make the new help icon clearer

### PR DESCRIPTION
**Pull request by lasjorg**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Just to get the ball rolling. I'm not sure if this is how we want it to look or not. Looking at the icons and their alignment has made me go blind.

Related issue: https://github.com/freeCodeCamp/freeCodeCamp/issues/49297

### CODE REVIEW

File: client/src/assets/icons/help.tsx [hunk 1]
```diff
@@ -3,18 +3,45 @@ import React from 'react';
...
+
+  const speechBubbleIcon = {
+    position: 'absolute'
+  } as React.CSSProperties;
+
+  const questionMarkIcon = {
+    position: 'absolute',
+    top: '-3px',
+    width: '9px'
+  } as React.CSSProperties;
```
Comment (Sboonny) [inline client/src/assets/icons/help.tsx:21]: having the values inside the component, make it reassigned every time the component render, can you move the values out of the help function.

**Comment by Sboonny** [comment]
> Looking at the icons and their alignment has made me go blind.

I actually can't see it 🤣, nice done it looks lovely 👍.

File: client/src/assets/icons/help.tsx (hunk 1 above)
Comment (Sboonny) [reply client/src/assets/icons/help.tsx:21]: I don't think there will be a need for the props, but let's see how they implement the changes first.

It isn't a big deal, I can create a quick PR after the issue is closed, if the props aren't accessed.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Review by naomi-lgbt — APPROVED** [review]
This icon should make things a little clearer (hopefully).

**Pull request merged**